include/deal.II/grid/reference_cell.h

```diff
@@ -1115,6 +1197,301 @@ ReferenceCell::faces_for_given_vertex(const unsigned int vertex) const
+             {{
+               {{8, 13, 16, 0, X, X}},
+               {{9, 12, 1, 21, X, X}},
+               {{10, 2, 17, 20, X, X}},
+               {{3, 14, 18, 22, X, X}},
+               {{11, 4, 0, 5, X, X}},
+               {{15, 4, 1, 6, X, X}},
+               {{19, 2, 5, 7, X, X}},
+               {{23, 6, 7, 3, X, X}},
+             }},
+             // new line is (4, 9)
+             {{
+               {{8, 13, 16, 0, X, X}},
+               {{9, 12, 1, 21, X, X}},
+               {{10, 2, 17, 20, X, X}},
+               {{3, 14, 18, 22, X, X}},
+               {{11, 4, 5, 2, X, X}},
+               {{15, 6, 7, 3, X, X}},
+               {{19, 5, 0, 6, X, X}},
+               {{23, 1, 4, 7, X, X}},
+             }}}};
+          return cell_quads_tet[refienement_choice];
+        }
+
+      case ReferenceCells::Hexahedron:
+        {
+          constexpr dealii::ndarray<unsigned int, 8, 6> cell_quads_hex = {{
+            {{12, 0, 20, 4, 28, 8}},  // bottom children
+            {{0, 16, 22, 6, 29, 9}},  //
+            {{13, 1, 4, 24, 30, 10}}, //
+            {{1, 17, 6, 26, 31, 11}}, //
+            {{14, 2, 21, 5, 8, 32}},  // top children
+            {{2, 18, 23, 7, 9, 33}},  //
+            {{15, 3, 5, 25, 10, 34}}, //
+            {{3, 19, 7, 27, 11, 35}}  //
+          }};
+          return cell_quads_hex;
+        }
+      default:
+        DEAL_II_NOT_IMPLEMENTED();
+    }
+
+  return {};
+}
+
+
+
+constexpr dealii::ndarray<unsigned int, 8, 4>
+ReferenceCell::new_isotropic_child_cell_vertices(
+  const unsigned int refienement_choice) const
+{
+  // AssertIndexRange(refienement_choice, n_isotropic_refinement_choices());
+
+  switch (this->kind)
+    {
+      case ReferenceCells::Tetrahedron:
+        {
+          constexpr dealii::ndarray<unsigned int, 3, 8, 4> cell_vertices_tet = {
+            {// new line is (6,8)
+             {{{{0, 4, 6, 7}},
+               {{4, 1, 5, 8}},
+               {{6, 5, 2, 9}},
+               {{7, 8, 9, 3}},
+               {{4, 5, 6, 8}},
+               {{4, 7, 8, 6}},
+               {{6, 9, 7, 8}},
+               {{5, 8, 9, 6}}}},
+             // new line is (5,7)
+             {{{{0, 4, 6, 7}},
+               {{4, 1, 5, 8}},
+               {{6, 5, 2, 9}},
+               {{7, 8, 9, 3}},
+               {{4, 5, 6, 7}},
+               {{4, 7, 8, 5}},
+               {{6, 9, 7, 5}},
+               {{5, 8, 9, 7}}}},
+             // new line is (4,9)
+             {{{{0, 4, 6, 7}},
+               {{4, 1, 5, 8}},
+               {{6, 5, 2, 9}},
+               {{7, 8, 9, 3}},
+               {{4, 5, 6, 9}},
+               {{4, 7, 8, 9}},
+               {{6, 9, 7, 4}},
+               {{5, 8, 9, 4}}}}}};
+          return cell_vertices_tet[refienement_choice];
+        }
+      default:
+        DEAL_II_NOT_IMPLEMENTED();
+    }
+
+  return {};
+}
+
+
+
+constexpr dealii::ndarray<unsigned int, 4, 3>
+ReferenceCell::reference_cell_face_vertices() const
```

Comment:
This info is already part of `ReferenceCell`.

Reply:
Removed the function, now use `ReferenceCell::face_to_cell_vertices()` in `tria.cc` instead.